### PR TITLE
Add option to hide recordings channel

### PR DESCRIPTION
**Pull request by joern-h**
As suggested in #16, i added an option to hide the recordings channel as the recordings are also accessible through the LiveTV section..

I successfully tested it with jellyfin 10.6.4

### CODE REVIEW

**Comment by joern-h** [comment]
Sorry for all the force pushes, i am not familiar with editing html / javascript.

It should be done now.

**Review by crobibero — APPROVED** [review]

**Comment by crobibero** [comment]
Also please bump plugin version found [here](https://github.com/jellyfin/jellyfin-plugin-tvheadend/blob/769330742e328b0b9ee2a708d69778c5d2aa2e64/build.yaml#L4) and [here](https://github.com/jellyfin/jellyfin-plugin-tvheadend/blob/master/TVHeadEnd/TVHeadEnd.csproj#L5-L6)

**Comment by jakobkukla** [comment]
Is there something that prevents you from merging this? I would love to have the option to disable that library :)

File: TVHeadEnd/TVHeadEnd.csproj [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <AssemblyVersion>8.0.0.0</AssemblyVersion>
-    <FileVersion>8.0.0.0</FileVersion>
+    <AssemblyVersion>8.1.0.0</AssemblyVersion>
+    <FileVersion>8.1.0.0</FileVersion>
```
Comment (h1dden-da3m0n) [inline TVHeadEnd/TVHeadEnd.csproj:6]: The Jellyfin Plugins only use the major version for their releases.
```suggestion
    <AssemblyVersion>9.0.0.0</AssemblyVersion>
    <FileVersion>9.0.0.0</FileVersion>
```

File: build.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 name: "TVHeadend"
 guid: "3fd018e5-5e78-4e58-b280-a0c068febee0"
-version: "8.0.0.0"
+version: "8.1.0.0"
```
Comment (h1dden-da3m0n) [inline build.yaml:4]:
```suggestion
version: "9.0.0.0"
```

**Review by h1dden-da3m0n — CHANGES_REQUESTED** [review]
otherwise I think this PR should still be :+1:

File: TVHeadEnd/TVHeadEnd.csproj (hunk 1 above)
Comment (oddstr13) [reply TVHeadEnd/TVHeadEnd.csproj:6]: Version has already been bumped, but no release was cut for it.

**Review by oddstr13 — APPROVED** [review]

**Comment by oddstr13** [comment]
Version was bumped in a different PR, but no release has been made since. I force-pushed away the version bumps.

**Review by h1dden-da3m0n — APPROVED** [review]

**Pull request merged**